server/controllers/stock/reports/stock/exit/exitToLoss.js

```diff
@@ -0,0 +1,37 @@
+const db = require('../../../../../lib/db');
+
+/**
+ * @function fetch
+ * @description fetch stock exit for patient
+ */
+function fetch(depotUuid, dateFrom, dateTo, showDetails) {
+  const IS_EXIT = 1;
+  const EXIT_TO_LOSS_ID = 11;
+  const sql = `
+  SELECT 
+    i.code, i.text, iu.text AS unit_text, BUID(m.document_uuid) AS document_uuid,
```

Comment:
Hmm... Besides the `EXIT_TO_LOSS_ID` variable, what has changed between this `fetch()` function and the others?  Can they be combined?